fastapi_etebase/routers/authentication.py

```diff
@@ -13,28 +13,47 @@
 from pydantic import BaseModel
 
 from django_etebase import app_settings
+from django_etebase.models import UserInfo
 from django_etebase.serializers import UserSerializer
 from django_etebase.token_auth.models import AuthToken
 from django_etebase.views import msgpack_encode, msgpack_decode
 from ..fastapi_msgpack import MsgpackResponse, MsgpackRoute
-from ..token_auth import get_auth_data, get_login_user, LoginChallengeData, AuthData
+from ..token_auth import get_auth_data, get_login_user, LoginChallengeData, AuthData, get_authenticated_user
 
 authentication_router = APIRouter(route_class=MsgpackRoute)
 User = get_user_model()
 
 
-class AuthenticationLoginSerializer(BaseModel):
-    response: bytes
-    signature: bytes
-
-
-class AuthenticationLoginInnerSerializer(BaseModel):
+class LoginResponse(BaseModel):
     username: str
     challenge: bytes
     host: str
     action: Literal["login", "changePassword"]
 
 
+class Login(BaseModel):
+    response: bytes
+    signature: bytes
+
+    @property
```

Comment:
Maybe it should be `@cached_property` instead? (it's a django thing)

Reply:
good idea ill change it to use [cached_property](https://docs.python.org/3/library/functools.html#functools.cached_property)